src/Tribe/Dependency.php

```diff
@@ -548,6 +548,28 @@ protected function check_addon_dependencies( $main_class ) {
 		public static function instance() {
 			return tribe( self::class );
 		}
+
+		/**
+		 * Return if we have a premium plugin active.
+		 * Results may be unreliable if called before the tribe_plugins_loaded hook.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool
+		 */
+		public function has_active_premium_plugin(): bool {
```

Comment:
```suggestion
		public function has_active_not_premium_plugin(): bool {
```

Not premium?

Reply:
The naming is hard here - it checks both ET and TEC... `base` plugins?

Reply:
I think we should retain the name since the method only returns true in the event that there is at least one active plugin that isn't TEC or ET.